src/components/FactionWheel.vue

```diff
@@ -50,8 +41,27 @@ export default class FactionWheel extends Vue {
       .length;
   }
 
+  strokeWidth (pos: number) {
+    const planet = this.planet(pos);
+    if (this.gameData.players.some(p => p.faction && factions[p.faction].planet === planet)) {
+      return "0.2px";
+    }
+
+    return "0.05px";
+  }
+
   planet (pos: number) {
     const list = [Planet.Terra, Planet.Oxide, Planet.Volcanic, Planet.Desert, Planet.Swamp, Planet.Titanium, Planet.Ice, Planet.Gaia, Planet.Transdim];
+    if (pos < 7) {
+      const data = this.gameData;
+      const faction = data.player(data.currentPlayer).faction;
```

Comment:
This selects the current player - not the own player.

The own player is in `this.$store.state.gaiaViewer.player`. Maybe you can use `this.$store.state.gaiaViewer.player ?? data.currentPlayer` so that spectators have the same view as the current player.